crm_compassion/models/crm_lead.py

```diff
@@ -54,6 +55,16 @@ def create_event(self):
             "context": context,
         }
 
+    @api.multi
+    @api.depends("event_ids")
```

Comment:
Trigger recomputation as well if we change the planned sponsorships in a linked event.
```suggestion
    @api.depends("event_ids", "event_ids.planned_sponsorships")
```